.github/workflows/build.yml

```diff
@@ -12,7 +12,6 @@ jobs:
       fail-fast: false
       matrix:
         node:
-          - '10'
```

Comment:
Dropped node10 support as all our lambdas are on [node12](https://github.com/lessonnine/babbel.infrastructure/search?q=nodejs10) and as I have used `Promise.allSettled` function which is available from [node12](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled)